src/features/smartSearch/l10n/messageIds.ts

```diff
@@ -136,6 +140,63 @@ export default makeMessages('feat.smartSearch', {
         ),
       },
     },
+    emailBlacklist: {
+      inputString: m<{
+        addRemoveSelect: ReactElement;
+        reasonSelect: ReactElement;
+      }>(
+        '{addRemoveSelect} people who will not receive email because {reasonSelect}'
+      ),
+      reasonSelect: {
+        any: m('of any reason'),
+        unsubOrg: m('they have unsubscribed'),
+      },
+    },
+    emailClick: {
+      inputString: m<{
+        addRemoveSelect: ReactElement;
+        emailScopeSelect: ReactElement;
+        emailSelect: ReactElement | null;
+        linkScopeSelect: ReactElement;
+        linkSelect: ReactElement | null;
+        operatorSelect: ReactElement;
+        projectSelect: ReactElement | null;
+        timeFrame: ReactElement;
+      }>(
+        '{addRemoveSelect} people who have {operatorSelect} {linkScopeSelect} in {emailScopeSelect} {emailSelect} {projectSelect} {timeFrame} {linkSelect}'
+      ),
+      linkScopeSelect: {
+        anyFollowingLinks: m('any of the following links'),
+        anyLink: m('any link'),
+      },
+      operatorSelect: {
+        clicked: m('clicked'),
+        notClicked: m('not clicked'),
+      },
+    },
+    emailHistory: {
+      inputString: m<{
+        addRemoveSelect: ReactElement;
+        emailScopeSelect: ReactElement;
+        emailSelect: ReactElement | null;
+        operatorSelect: ReactElement;
+        projectSelect: ReactElement | null;
+        timeFrame: ReactElement;
+      }>(
+        '{addRemoveSelect} people who have {operatorSelect} {emailScopeSelect} {emailSelect} {projectSelect} {timeFrame}'
+      ),
+      operatorSelect: {
+        notOpened: m('not opened'),
+        notSent: m('not been sent'),
+        opened: m('opened'),
+        sent: m('been sent'),
```

Comment:
Here's yet another alternative: Replace the `notOpened` here with `not_opened` and `notSent` with `not_sent`, and you wouldn't have to do any conversion at all in the code that's using these translations.

We do try to use camelCase here in general, but there are examples of where we use snake_case because the keys are keywords coming from the API, and I think that's ok.